packages/cli/src/bin/index.ts

```diff
@@ -120,6 +123,20 @@ switch (command) {
     await codemod.default(values.transform!, values.src!, values.typescript);
     break;
   }
+
+  case 'patch-compat-table': {
+    const patchesPath = path.relative(process.cwd(), fileURLToPath(new URL('../../patches', import.meta.url)));
```

Comment:
we can just use `relative` from the already imported `node:path` module (line 2)